week1/shop/myshop.js

```diff
@@ -126,29 +129,33 @@ const allHashTagWrapper = document.createElement("ul");
 const moreHashtagBtn = document.querySelectorAll(".menu__hashtag > button");
 allHashTagWrapper.className = "menu_allHashtag";
 
+const handleMoreHashtag = (wrapper, flag) => {
+  flag = !flag;
+
+  if (flag) {
+    wrapper.className = "menu_allHashtag";
+    menuItems[i].prepend(wrapper);
+
+    shopData[i].hashtags.forEach((el) => {
+      const hashtagItem = document.createElement("li");
+      hashtagItem.innerText = el;
+      wrapper.appendChild(hashtagItem);
+    });
+  } else {
+    while (wrapper.firstChild) {
+      wrapper.firstChild.remove();
+    }
+    wrapper.remove();
```

Comment:
다 지우는거면!!
```suggestion
wrapper.innerText = '';
```
하면 한번에 델텐데용!